src/verify/reducer.ts

```diff
@@ -1,386 +1,43 @@
-  EnsuringRealHumanUser = 'EnsuringRealHumanUser',
-  StartingKomenciSession = 'StartingKomenciSession',
-  FetchingPhoneNumberDetails = 'FetchingPhoneNumberDetails',
-  FetchingMtw = 'FetchingMtw',
-  FetchingOnChainData = 'FetchingOnChainData',
-  Error = 'Error',
-}
-
-// Idle State
-interface Idle {
-  type: StateType.Idle
-}
-export const idle = (): Idle => ({ type: StateType.Idle })
-
-// PreparingKomenci State
-interface Preparing {
-  type: StateType.Preparing
-}
-// {}: Omit<PreparingKomenci, 'type'>
-export const preparing = (): Preparing => ({
-  type: StateType.Preparing,
-})
-
-// EnsuringRealHumanUser State
-interface EnsuringRealHumanUser {
-  type: StateType.EnsuringRealHumanUser
-}
-export const ensuringRealHumanUser = (): EnsuringRealHumanUser => ({
-  type: StateType.EnsuringRealHumanUser,
-})
-
-// StartingKomenciSession State
-interface StartingKomenciSession {
-  type: StateType.StartingKomenciSession
-}
-export const startingKomenciSession = (): StartingKomenciSession => ({
-  type: StateType.StartingKomenciSession,
-})
-
-// FetchingPhoneNumberDetails State
-interface FetchingPhoneNumberDetails {
-  type: StateType.FetchingPhoneNumberDetails
-}
-export const fetchingPhoneNumberDetails = (): FetchingPhoneNumberDetails => ({
-  type: StateType.FetchingPhoneNumberDetails,
-})
-
-// FetchingMtw State
-interface FetchingMtw {
-  type: StateType.FetchingMtw
-}
-export const fetchingMtw = (): FetchingMtw => ({
-  type: StateType.FetchingMtw,
-})
-
-// FetchingVerificationOnChain State
-interface FetchingOnChainData {
-  type: StateType.FetchingOnChainData
-}
-export const fetchingOnChainData = (): FetchingOnChainData => ({
-  type: StateType.FetchingOnChainData,
-})
-
-// Error State
-interface Error {
-  type: StateType.Error
-  message: string
-}
-export const error = (message: string): Error => ({
-  type: StateType.Error,
-  message,
-})
-
-type InternalState =
-  | Idle
-  | Preparing
-  | EnsuringRealHumanUser
-  | StartingKomenciSession
-  | FetchingPhoneNumberDetails
-  | FetchingMtw
-  | FetchingOnChainData
-  | Error
-
 export interface KomenciContext {
-  errorTimestamps: number[]
   unverifiedMtwAddress: string | null
-  sessionActive: boolean
-  sessionToken: string
-  callbackUrl: string | undefined
-  captchaToken: string
-}
-
-export enum KomenciAvailable {
-  Yes = 'YES',
-  No = 'NO',
-  Unknown = 'UNKNOWN',
 }
 
 export interface State {
-  status: AttestationsStatus & { komenci: boolean }
-  actionableAttestations: ActionableAttestation[]
-  currentState: InternalState
   komenci: KomenciContext
-  komenciAvailable: KomenciAvailable
-  phoneHash?: string
-  e164Number?: string
-  retries: number
-  withoutRevealing: boolean
-  TEMPORARY_override_withoutVerification?: boolean
-  // KomenciKit configuration
-  komenciConfig: {
-    useLightProxy: boolean
-    allowedDeployers: string[]
-  }
 }
 
 const initialState: State = {
   komenci: {
-    errorTimestamps: [],
     unverifiedMtwAddress: null,
```

Comment:
this guy is used in the `runVerificationMigration` saga, i wasn't 100% sure that we want to remove that functionality. although it's really old :/

Reply:
Cool I was about to ask if we could remove this property too. We can safely remove this migration now.

Reply:
oh fab, then i'll remove it then merge! one less redux store 🥳